Androidcode/App/app/src/main/java/com/example/christopher/myapplication/ImageServer.java

```diff
@@ -0,0 +1,122 @@
+package com.example.christopher.myapplication;
+
+import org.opencv.core.Mat;
+import org.usfirst.frc.team1885.visioncode.utils.SimpleImage;
+
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.io.OutputStream;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.LinkedBlockingDeque;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.LinkedTransferQueue;
+
+/**
+ * Created by Atishay on 1/29/2017.
+ */
+
+public class ImageServer {
+
+
+    ImageServer() {
+
+    }
+
+    public void  connect() {
+        final ExecutorService service = Executors.newCachedThreadPool();
```

Comment:
Please name of the thread by 

    Executors.newCachedThreadPool(new ThreadFactory() {						
        @Overrid					
        public Thread newThread(Runnable r) {
	    return new Thread("Name",r );
	}
	});